provision-staging.sh

```diff
@@ -29,7 +29,7 @@ ArtifactSourceBucketEventTriggerRoleArn=${CFN_frontend_pipeline_ArtifactPromotio
 # ------------------------------
 export AWS_ACCOUNT=di-authentication-staging
 export AWS_PROFILE=di-authentication-staging-AWSAdministratorAccess
-aws sso login --profile "${AWS_PROFILE}"
+# aws sso login --profile "${AWS_PROFILE}"
```

Comment:
want this commented?

Reply:
@dbes-gds yes. Duplicate line to line 11.